Makefile

```diff
@@ -144,7 +149,8 @@ python-example: ## Run a specific Python example. Usage: make python-example exa
 %:
 	@true
 python-example:
-	PYTHONPATH=bindings/python python3 bindings/python/examples/$(word 2,$(MAKECMDGOALS)).py
+	@echo "\nRunning Python example \"$(word 2,$(MAKECMDGOALS))\""
```

Comment:
for me it's not applying the newline character in the fish shell but instead literally writing `\n`

Reply:
Time to switch to another shell :trollface: Works fine for me with zsh
Does it work for you with `@printf` instead of `@echo`?

Reply:
I'll try

Reply:
yeah that works. Seems like this is the most shell independent solution. I'll push the change to this branch, and then please try if it still works for you.

Reply:
Works fine